rpc/models.py

```diff
@@ -326,6 +326,8 @@ class Meta:
     def __str__(self):
         return str(self.number)
 
+    history = HistoricalRecords()
```

Comment:
style nit: should move up to be with the other fields ahead of the `__str__` method